app/Policies/WebhookPolicy.php

```diff
@@ -11,30 +11,29 @@ class WebhookPolicy
 {
     use HandlesAuthorization;
 
-    /**
-     * Determine whether the user can update the webhook.
-     *
-     * @param  \App\Models\User  $user
-     * @param  \App\Webhook  $webhook
-     * @return mixed
-     */
-    public function update(User $user, Webhook $webhook)
-    {
-        return $user->id === $webhook->user_id;
-    }
+    /**	
+     * Determine whether the user can update the webhook.	
+     *	
+     * @param  \App\Models\User  $user	
+     * @param  \App\Webhook  $webhook	
+     * @return mixed	
+     */	
+    public function update(User $user, Webhook $webhook)	
+    {	
+        return $user->id === $webhook->user_id;	
+    }	
 
-    /**
-     * Determine whether the user can create the webhook.
-     *
-     * @param  \App\Models\User  $user
-     * @param  \App\Webhook  $webhook
-     * @return mixed
-     */
-    public function create(User $user)
-    {
-        return $user->role === UserType::USER;
+    /**	
+     * Determine whether the user can create the webhook.	
+     *	
+     * @param  \App\Models\User  $user	
+     * @param  \App\Webhook  $webhook	
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Policies/WebhookPolicy.php</br>**Line**: 30</br>Whitespace found at end of line